OpenRA.Mods.RA/OpenRA.Mods.RA.csproj

```diff
@@ -80,7 +81,7 @@
     <Compile Include="Activities\Infiltrate.cs" />
     <Compile Include="Activities\LayMines.cs" />
     <Compile Include="Activities\Leap.cs" />
-    <Compile Include="Activities\RAHarvesterDockSequence.cs" />
+    <Compile Include="Activities\HarvesterDockSequence.cs" />
```

Comment:
No, this will stay here for the moment. I've promised to move all the Harvester- and Refinery-related files from Mods.RA to Mods.Common when this is done anyway, so I'll just do it at once.